src/main/java/com/epam/tc/hw7/entities/User.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.hw7.entities;
+
+public class User {
+    public static final User ROMAN = new User("Roman", "Jdi1234", "ROMAN IOVLEV");
```

Comment:
It is not good practice for DTO to store self constants
Please move it in other place

Reply:
Fixed with GetProperties Utility